### ISSUE TITLE
PowerBI 508 - Meet with DME 

### ISSUE THREAD

**Issue by Maroyafaied** [ONRR/onrr.gov-site#2380]

**Comment by Maroyafaied** [comment]
@eelzi-ONRR  we have a meeting at 1:30PM today with the contractors for the Data Management Environment. Can you please gather the list of links/resources we chatted about in slack? We can discuss this during our check in as well.

**Comment by eelzi-ONRR** [comment]
@Maroyafaied I'm going to spend some time before the meeting making this into a prettier resource list we can share, but here's all the links just for our own reference:

- https://www.highcharts.com/blog/tutorials/10-guidelines-for-dataviz-accessibility/

- https://learn.microsoft.com/en-us/power-bi/create-reports/desktop-accessibility-creating-reports

- https://doimspp.sharepoint.com/sites/NRRD/Shared%20Documents/Forms/AllItems.aspx?id=%2Fsites%2FNRRD%2FShared%20Documents%2FGeneral%2FONRR%2Egov%2FDesign%2FVisual%20Design%2Fstyle%2Etile%2Epdf&parent=%2Fsites%2FNRRD%2FShared%20Documents%2FGeneral%2FONRR%2Egov%2FDesign%2FVisual%20Design

- https://doi.gov/ocio/section508/references

- https://github.com/ONRR/onrr.gov-site/wiki/Content-guide

- https://blog-nrrd.doi.gov/four-part-hybrid-testing/

- https://blog-nrrd.doi.gov/beyond-auto/

- https://blog-nrrd.doi.gov/accessibility/

- https://github.com/ONRR/onrr.gov-site/wiki/Accessibility-Policy

- https://github.com/ONRR/onrr.gov-site/wiki/Accessibility

I also found this report/guide on accessible data visualizations that was just released earlier this month. It's pretty long - I'll add the link to the resources list. I'll also pull out key points that could be useful for them and maybe for us to add to the wiki.

- https://www.urban.org/sites/default/files/2022-12/Do%20No%20Harm%20Guide%20Centering%20Accessibility%20in%20Data%20Visualization.pdf

**Comment by Maroyafaied** [comment]
@eelzi-ONRR created a doc with resources and we met with them. They will let us know if they need us to connect them with screen reader users to test the templates.  https://doimspp-my.sharepoint.com/:w:/g/personal/elzie_mms_gov/EcZGebt7p2tAvvZ-4dair8IBvCX-HfkIbItHCo4kOSf8FA
